envs/public/lookandfeel/setting.json

```diff
@@ -0,0 +1,13 @@
+{
+  "headerBgColor": {
+    "darkMode": "",
+    "lightMode": ""
+  },
+  "loginBgImg": "",
+  "loginTitle": "SCS Registry",
+  "product": {
+    "name": "SCS Registry",
+    "logo": "SCS_logo.png",
+    "introduction": "Sovereign Cloud Stack (SCS) Registry - deployment of Harbor open source (https://registry.scs.community/license) registry for the SCS community usage. The main features include proxying and caching major repositories like docker.io, ghcr.io, quay.io and others. It runs daily security scans and maintains cleanliness through automated garbage collection and log rotation. For enhanced access control, it offers OIDC Authentication via SCS UCS. Default storage quotas are 100 GiB per project. Other additional features are webhooks, automatic onboarding, and content signing/validation. Feel free to explore and contribute to the https://github.com/SovereignCloudStack/k8s-harbor repository."
```

Comment:
Consider reformulating the first sentence: e.g.:
The Sovereign Cloud Stack (SCS) Registry represents a reference deployment of the SCS registry, built upon the open-source Harbor software, intended for the benefit of the SCS community.